Source/WebKit/UIProcess/Extensions/WebExtensionContext.h

```diff
@@ -291,6 +295,14 @@ class WebExtensionContext : public API::ObjectImpl<API::Object::Type::WebExtensi
     void addListener(WebPageProxyIdentifier, WebExtensionEventListenerType);
     void removeListener(WebPageProxyIdentifier, WebExtensionEventListenerType);
 
+    // Permissions APIs
+    void getAll(WebPageProxyIdentifier, CompletionHandler<void(Vector<String> permissions, Vector<String> origins)>&&);
```

Comment:
We will run into namespace clashes with the methods names at some point. We should prefix the messages and the methods with the namespace, so `permissionsGetAll()`. Like I did with `test` above.